store/store_test.go

```diff
@@ -198,7 +198,7 @@ func TestGetImageManifest(t *testing.T) {
 func TestGetAci(t *testing.T) {
 	type test struct {
 		name     types.ACName
-		labels   types.Labels
+		labels   []types.Label
```

Comment:
JFYI, the reason I've done this change is to make go vet happy. `go vet` is currently uncapable of deducting that `types.Labels` is in fact array type and thinks that `{}` is using unnamed struct fields. 

The current `./test` does not catch this for some reason